e-commerce_fake-store/src/pages/home-page/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useCallback } from "react";
+import Product from "../../components/product-card";
+import Navbar from "../../components/navbar";
+// import {useHistory } from "react-router-dom";
+
+const Home = () => {
+  const [allProducts, setAllProducts] = React.useState([]);
+  const [search, setSearch] = React.useState("");
+  // const history = useHistory();
+
+  // api varibale
+  const api = "https://fakestoreapi.com/";
+
+  const getAllProducts = useCallback(async () => {
+    const response = await fetch(`${api}` + "products");
+    const data = await response.json();
+    setAllProducts(data);
+  }, []);
```

Comment:
Use the proper string interpolation here.

``` await fetch(`${api}/products`) ```

@shubhamshukla91

Reply:
okay.